### DISCUSSION TITLE
Include accuracy in the ipc.txt files

### DISCUSSION THREAD

**Discussion by DerHamm3r** [ppy/osu#20084] [Ideas]
It would be cool if cutting edge was able to also write player/team accuracies in either an existing or a new ipc file. The reason for that would be that you could modify the lazer tournament client to display accuracy instead of score in case a tournament uses accuracy as the victory condition. (Having the option in the lazer client to display accuracy without modifying it would also be a cool feature for the future, but for now the change to cutting edge would be enough to get it to work)

**Comment by peppy** [comment]
Sounds cool, but I think before then we'll have the full lazer tournament ecosystem online which should remove the whole `ipc.txt` communication in the first place.

If you are running a large tournament which requires this, please let us know though. We might be able to expedite it.

**Reply by DerHamm3r** [reply]
There is one going on at the moment, which is why I decided to make the suggestions here.
https://osu.ppy.sh/community/forums/topics/1619012?n=1
It is in RO32 this weekend, they are trying to get it badged, but it is up to you to decide if that counts as a "large" tournament, as well as if it can be done in time. If the implementation would take too long it probably is not worth it.
